cpg-neo4j/src/main/kotlin/de/fraunhofer/aisec/cpg_vis_neo4j/Application.kt

```diff
@@ -334,8 +346,13 @@ class Application : Callable<Int> {
             translationConfiguration.sourceLocations(filePaths)
         }
 
-        if (!noDefaultPasses) {
+        if (!noDefaultPasses && customPasses == "ALL") {
```

Comment:
Why will "ALL" result in the default passes and not in really all available passes?

Reply:
Good point. Renamed it to "DEFAULT".